File: haskell/iniciante/1010.hs
-- title: Cálculo Simples
-- number of the problem: 1010
-- date: 20/09/20
-- author: ltsdw

import Text.Printf

total2Pay :: (Double, Double , Double, Double) -> Double
total2Pay (qtd1, price1, qtd2, price2) = (qtd1 * price1) + (qtd2 * price2) 

printTotal :: Double -> IO ()
printTotal total = do
    printf "VALOR A PAGAR: R$ %.2f\n" total

main :: IO ()
main = do
    l1 <- getLine
    l2 <- getLine

    let ( _ : qtd1 : price1: _ ) = map (read :: String -> Double) $ words l1
    let ( _ : qtd2 : price2: _ ) = map (read :: String -> Double) $ words l2

    let total = total2Pay (qtd1, price1, qtd2, price2)

    printTotal total

File: python/iniciante/1010.py
'''
title: Cálculo Simples
number of the problem: 1010
date: 27/02/2018
author: ltsdw
'''

_, qnt1, prec1 = map(float, input('').split())
_, qnt2, prec2 = map(float, input('').split())

valor = (qnt1 * prec1) + (qnt2 * prec2)

print(f'VALOR A PAGAR: R$ {valor:.2f}')

